### PR TITLE
up installation documentation

### PR DESCRIPTION
**Pull request by buixor**
- [x] update installation documentation to make note of official debian repos and our debian repos
 - [x] pimp docker setup page
 - [x] reference freebsd ports : notify that redhat/centOS and freebsd are wip

### CODE REVIEW

**Comment by zorglube** [comment]
Hello,

I have stared some work in order to install CrowdSec (CS) on my FreeNAS (FreeBSD). 
As I see here you'll make some work about install CS on FreeBSD, thus I very interested, especially about the `Wizard` & `Metabase` parts. 

Side question, are you thinking about making an official FreeNAS/TrueNAS plugin ? 

BR, 
Zorglube

**Comment by buixor** [comment]
Hello @zorglube !

I don't have much knowledge about freeNAS/trueNAS, but as long as the effort is reasonable, it's definitely something we can consider :slightly_smiling_face: 

If you don't mind, can you open an issue describing what kind of features/use-cases you would expect ?

cheers,

File: docs/v1.X/docs/getting_started/installation.md [hunk 1]
```diff
@@ -1,4 +1,80 @@
...
+ - Some users download the DEB package directly and [install it manually](/Crowdsec/v1/getting_started/installation/#manually-install-the-debian-package)
+ - Some users download the tarball directly and [install it manually](/Crowdsec/v1/getting_started/installation/#install-from-release-tarball)
+ - Some users use the [docker hub image](https://hub.docker.com/r/crowdsecurity/crowdsec)
+ - And the most adventurous might want to [build & install from source](/Crowdsec/v1/getting_started/installation/#install-from-source)
+ - And some might even want to [build their own docker image](/Crowdsec/v1/getting_started/installation/#build-docker-image)
+ - Or use it with [docker-compose](https://github.com/crowdsecurity/example-docker-compose)
+
+
+!!! info
+    Packaging for FreeBSD and RedHat/CentOS are WIP at the time of writting. Documentation will be updated once those packages are published & functional.
```
Comment (registergoofy) [inline docs/v1.X/docs/getting_started/installation.md:17]: writing

File: docs/v1.X/docs/getting_started/installation.md [hunk 2]
```diff
@@ -1,4 +1,80 @@
...
+
+# Install using crowdsec repository
+
+Crowdsec distributes their own pragmatic debian packages that closely follow the developpement stream (packages are automatically published on release), and are suitable for those that want to keep up with latest changes of crowdsec.
```
Comment (registergoofy) [inline docs/v1.X/docs/getting_started/installation.md:21]: development

File: docs/v1.X/docs/getting_started/installation.md [hunk 3]
```diff
@@ -1,4 +1,80 @@
...
+</details>
+
+```bash
+wget -qO - https://s3-eu-west-1.amazonaws.com/crowdsec.debian.pragmatic/crowdsec.asc |sudo apt-key add - && sudo apt-add-repository "https://s3-eu-west-1.amazonaws.com/crowdsec.debian.pragmatic/$(lsb_release -cs) $(lsb_release -cs) main"
+sudo apt-get update
+```
+
+The following debian suites / architectures are available :
+
+## install crowdsec
```
Comment (registergoofy) [inline docs/v1.X/docs/getting_started/installation.md:39]: install crowdsec should go after compatibility matrix ?

File: docs/v1.X/docs/getting_started/installation.md [hunk 4]
```diff
@@ -1,4 +1,80 @@
...
+
+```bash
+sudo apt-get install crowdsec
+```
+
+## compatbility matrix
```
Comment (registergoofy) [inline docs/v1.X/docs/getting_started/installation.md:45]: compatibility

File: docs/v1.X/docs/getting_started/installation.md [hunk 5]
```diff
@@ -1,4 +1,80 @@
...
+|  bionic | amd64, arm64, i386   |
+| bullseye   | amd64, i386 |
+| buster | amd64, arm64, i386 |
+| focal |  amd64, arm64, i386 | 
+| sid | amd64, i386 |
+| stretch | amd64, arm64, i386 | 
+| xenial | amd64, arm64, i386 | 
+
+</center>
+
```
Comment (registergoofy) [inline docs/v1.X/docs/getting_started/installation.md:60]: Maybe add that missing build are wip too. We want to add debian packages for armhf and raspberrypios too.

**Review by registergoofy — APPROVED** [review]
few mispells

File: docs/v1.X/docs/getting_started/installation.md (hunk 5 above)
Comment (buixor) [reply docs/v1.X/docs/getting_started/installation.md:60]: I think this is true for more advanced users

**Comment by zorglube** [comment]
> Hello @zorglube !
> 
> I don't have much knowledge about freeNAS/trueNAS, but as long as the effort is reasonable, it's definitely something we can consider :slightly_smiling_face: 
> 
> If you don't mind, can you open an issue describing what kind of features/use-cases you would expect ?
> 
> cheers,
> 
> 

At first I expect an FreeBSD port fully working, and and for the FreeNAS plugin I can handle it. 

Today the part that cause me a lot of issues are the : 
 - BSD make, on BSD I have to use `gmake` ; 
 - the `wizard.sh` have few issues with `install` ; 
 - the `Metabase`, since `Docker` is broken on BSD, I'm still working on that ;  

Cheers, 
Zorglube

**Pull request merged**